src/main/java/com/mparticle/kits/RadarKit.java

```diff
@@ -123,10 +130,24 @@ public void removeUserAttribute(String s) {
 
     @Override
     public void setUserIdentity(MParticle.IdentityType identityType, String id) {
-        if (identityType.equals(MParticle.IdentityType.CustomerId)) {
+        if (identityType.equals(MParticle.IdentityType.Alias)) {
+            Radar.reidentifyUser(id);
+            
+            Map<MParticle.IdentityType, String> identities = getUserIdentities();
+            String customerId = identities.get(MParticle.IdentityType.CustomerId);
+            if (customerId != null) {
+                Radar.setUserId(customerId);
+            }
```

Comment:
No need to check for CustomerId here, it will either a) already be present in `onKitCreate()` or it will be passed in `setUserIdentity() with the IdentityType == CustomerId

Other than that, this looks all good!